src/host/usbh.c

```diff
@@ -1362,6 +1362,8 @@ static void process_enumeration(tuh_xfer_t* xfer)
       enum_full_complete();
     break;
   }
+  // If required invalidate data cache after DMA transfer complete - see hcd.h for details.
+  hcd_dcache_invalidate((uint32_t)_usbh_ctrl_buf, sizeof(_usbh_ctrl_buf));
```

Comment:
should be removed since I think invalidate should be done within the HCD (EHCI in this case) see below feedback in hcd_edpt_xfer().

Reply:
I'll check this too... 🙂

Reply:
it should be removed, we will do it in hcd layer.